tasmota/language/it-IT.h

```diff
@@ -56,54 +56,54 @@
 #define D_AP "AP"                    // Access Point
 #define D_AS "come"
 #define D_AUTO "AUTO"
-#define D_BLINK "Blink"
-#define D_BLINKOFF "BlinkOff"
+#define D_BLINK "Lampeggia"
+#define D_BLINKOFF "Lampeggia OFF"
 #define D_BOOT_COUNT "Numero di boot"
 #define D_BRIGHTLIGHT "Luminoso"
 #define D_BSSID "BSSId"
 #define D_BUTTON "Pulsante"
 #define D_BY "da"                    // Written by me
-#define D_BYTES "Bytes"
+#define D_BYTES "Byte"
```

Comment:
Bytes. Non Byte, é plurale.

Reply:
In Inglese il plurale è "Bytes" ma in italiano il plurale è "Byte".
Non esiste la s finale nel plurale in italiano.

Reply:
Porca paletta hai ragione!

Reply:
Logging ha la sua traduzione ed è quella indicata "registrazione":
Una delle regole fondamentali dell'italiano per favorire la comprensione è quella di evitare a meno che non sia possibile evitare termine inglesi (Logging).
Soprattutto se un termine italiano c'è.
Bisogna anche evitare il problema opposto (ON=acceso - OFF=spento).
In questo caso meglio l'inglese perché nel conteso acceso/spento non è contestualizzato.

Reply:
É sicuramente complicato e non esiste una formula che funzioni in tutti i casi. 
Prendi per esempio "garage", puoi tradurlo con "autorimessa" ma tutti lo chiamano garage. 
Allo stesso modo "Host" non lo traduci in "ospite" (o peggio "oste").